egs/fisher_english/s5/local/semisup/chain/tuning/run_tdnn_15k_j.sh

```diff
@@ -0,0 +1,203 @@
+#!/bin/bash
+
+# Copyright 2017  Vimal Manohar
+# Apache 2.0
+
+set -e
+
+# This is fisher chain recipe for training a model on a subset of around 15 hours.
+# This is similar to _d, but uses a speed-perturbed data for tree building
```

Comment:
Why do you keep two scripts _d and _j?  It is better to keep the better one.

Reply:
I removed _d. But there are two versions, one with UNK model using phone LM and the other without.